src/urlValidator.js

```diff
@@ -0,0 +1,37 @@
+const { REQUEST_EXAMPLE, AVALIABLE_STYLES } = require('./constants');
+
+function urlValidator(textToConvert, style) {
+  const noStyle = {
+    message: `"toCase" query param is required. ${REQUEST_EXAMPLE}.`,
+  };
+  const noTextToConvert = {
+    message: `Text to convert is required. ${REQUEST_EXAMPLE}.`,
+  };
+  const wrongStyle = {
+    message: `This case is not supported. Available cases: ${AVALIABLE_STYLES.join(', ')}.`,
+  };
```

Comment:
You can create just one constant object with all errors messages and use it, also move it outside function